README.md

```diff
@@ -24,28 +36,60 @@ Some recorders has a `format` configuration that can be used to format the outpu
 
 ### Print recorder
 
-Print the data received through the console using the string format configured in `format`.
+Print the data received through the console using the string format configured in `format`. `format`configuration accepts dynamic values.
+
+Example configuration:
 
-    {
-        "recorders": [{
-            "type": "print",
-            "config": {
-                "format": "{device_id} C {celsius} / F {fahrenheit}"
-            }
-        }]
-    }
+```json
+{
+    "recorders": [{
+        "type": "print",
+        "config": {
+            "format": "{device_id} C {celsius} / F {fahrenheit}"
+        }
+    }]
+}
+```
 
 ### File recorder
 
-This recorder will create a file for each sensor read using the device id of the sensor as file name. The files will be stored in the path configured in `container` with the extension specified in `extension`. The data read from the sensor will be write using the string format in `format`.
-
-    {
-        "recorders": [{
-            "type": "file",
-            "config": {
-                "format": "{timestamp},{celsius},C",
-                "container": "/temperature/",
-                "extension": ".log"
-            }
-        }]
-    }
+This recorder will create a file for each sensor read using the device id of the sensor as file name. The files will be stored in the path configured in `container` with the extension specified in `extension`. The data read from the sensor will be write using the string format in `format`. `format`configuration accepts dynamic values.
+
+Example configuration:
+
+```json
+{
+    "recorders": [{
+        "type": "file",
+        "config": {
+            "format": "{timestamp},{celsius},C",
+            "container": "/temperature/",
+            "extension": ".log"
+        }
+    }]
+}
+```
+
+### HTTP Recorder
+
+This recorder will send measured data as JSON object in the body to the configured `endpoint` using HTTP POST. The `endpoint` can contain dynamic values.
+
+Optionally `headers` can be configured to be sent in the request.
+
+> `Content-Type` header should not be provided and if so will not be used since it is fixed to `application/json`.
+
+Example configuration:
+
+```json
```

Comment:
Update config to show a valid example (`json_template` is missing and headers is no longer an array).